emails/DrawingEmail.tsx

```
@@ -0,0 +1,247 @@
+              className="mx-auto w-80"
+            />
+          </div>
+          <Text className="text-[16px] leading-[26px]">
+            Hi {userName},
+          </Text>
+          <Text className="text-[16px] leading-[26px]">
+            Your secret santa group drawing has completed! You matched with <strong>{recipient}</strong>.  See your AI generated gift suggestions below.
+          </Text>
+            {giftSuggestions?.length !== 0 && 
+              giftSuggestions.map((gift) => (
+                <Section style={{ width: '75%', marginBottom: '15px', marginTop: '15px' }}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)